src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -101,6 +101,7 @@ private static void updateAnimIndex()
    */
   public static BufferedImage buildCoPowerBar(Commander co, int[] abilityPoints, double currentPower, boolean leftSide)
   {
+    final double powerDrawScaleW = 4.0/Commander.CHARGERATIO_FUNDS;
```

Comment:
Magic 4?

Reply:
The two magic 2's were not multiplicative with each other. The new power bar is twice as long as the old one.

Reply:
You're correct, but I like that scaling better anyway. (I actually applied that change in the AWBW branch, it helps the shorter meters make a little more sense to look at)